From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tim Mullin <tim@cpanel.net>
Date: Fri, 2 Nov 2018 16:35:12 -0500
Subject: [PATCH 7/8] Fixed duplicate case value caused by libcurl 7.62

From curl.h:

588 /* added in 7.62.0 */
589 #define CURLE_SSL_CACERT CURLE_PEER_FAILED_VERIFICATION

This caused a compile error in a case statement that
specified both CURLE_SSL_CACERT and CURLE_PEER_FAILED_VERIFICATION
for a specific case.  In the previous versions of libcurl,
these were two distinct values.
---
 src/agent/Core/SecurityUpdateChecker.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/agent/Core/SecurityUpdateChecker.h b/src/agent/Core/SecurityUpdateChecker.h
index f3c4d10..46784bb 100644
--- a/src/agent/Core/SecurityUpdateChecker.h
+++ b/src/agent/Core/SecurityUpdateChecker.h
@@ -256,7 +256,6 @@ private:
 				error.append(" for proxy address " + sessionState.config["proxy_url"].asString());
 				break;
 
-			case CURLE_SSL_CACERT:
 				// Peer certificate cannot be authenticated with given / known CA certificates. This would happen
 				// for MITM but could also be a truststore issue.
 			case CURLE_PEER_FAILED_VERIFICATION:
